NodeTemplate/controllers/subscriberController.js

```diff
@@ -0,0 +1,51 @@
+const { validateAndDecodeJwt, updateRecordDescription } = require("../services");
+
+async function subscriberController(req, res) {
+  let message;
+
+  // get auth header from request
+  const authHeader = req.headers.authorization;
+  if (!authHeader) {
+    message = "Authorization header not found in the request";
+    console.error(message);
+    return res.status(200).json({ ProcessingStatus: -1, ResponsePayload: { message } });
```

Comment:
Should probably return 401 here, since the user is Unauthorized

Reply:
Profisee's webhook code will retry on non-OK responses, so even in the case of something going wrong we return a 200 and use the response payload with ProcessingStatus=-1 to indicate something went wrong. However, looking back at the Azure Function template, it does send a 401/403 for the first checks on the auth header and JWT. I'll update this template to do the same.